src/libs/PolicyUtils.ts

```diff
@@ -97,7 +97,7 @@ function shouldShowPolicy(policy: OnyxEntry<Policy>, isOffline: boolean): boolea
     );
 }
 
-function isExpensifyTeam(email: string): boolean {
+function isExpensifyTeam(email?: string): boolean {
```

Comment:
Since the email is always passed but can be undefined
```suggestion
function isExpensifyTeam(email: string | undefined): boolean {
```